<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SourceDialog</class>
 <widget class="QDialog" name="SourceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmallBASIC - Program Source</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sbasic.qrc">
    <normaloff>:/images/smallbasic.png</normaloff>:/images/smallbasic.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="plainText">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="sbasic.qrc"/>
 </resources>
 <connections/>
</ui>
